### PR TITLE
Add preview for textures in the inspector

### PR DESCRIPTION
**Pull request by Bloadrick**
I have added a preview for different format of textures in the inspector.

The DDS format is now render in a BabylonJS Canvas so that you can see it fully if it is a cube texture.

The feature request : https://github.com/BabylonJS/Babylon.js/issues/4589

### CODE REVIEW

File: inspector/src/tabs/TextureTab.ts [hunk 1]
```diff
@@ -196,4 +227,101 @@ export class TextureTab extends Tab {
         }
         item.active(true);
     }
+
+}
+
+class TestDDSPreview {
```
Comment (deltakosh) [inline inspector/src/tabs/TextureTab.ts]: Not fan of having test class here

Comment (Bloadrick) [reply inspector/src/tabs/TextureTab.ts]: Yes you are right and right, i'm fixing that asap.

I was not aware of the webGL context limit, I need to check that.

**Comment by Jaskar** [comment]
I'm taking care of that this week!

**Comment by deltakosh** [comment]
Still ok?

**Comment by Jaskar** [comment]
Sorry, I was on holidays.
I'm looking at this before Friday!

**Pull request closed without merging**